OpenTreeMap/otm-core
# Add canopy percent choropleth layer

**Test:**
* Checkout https://github.com/OpenTreeMap/otm-tiler/pull/109 and restart the tiler
* Rebuild static assets `./scripts/grunt.sh`
* Load "PA Tree Map" (or any instance with canopy data)
  * Set the canopy category `update treemap_instance set canopy_boundary_category ='Municipality' where url_name='patreemap';`
* Ensure that the "Regional Canopy Percentages" layer exists
* Ensure that the filter control appears when "Regional Canopy Percentages" layer is visible
* Ensure that altering the filter control updates the canopy layer
 
![screenshot 2016-06-08 at 17 03 08](https://cloud.githubusercontent.com/assets/43062/15910861/ea1ba34c-2d9a-11e6-846b-4761d4972d92.png)


Connects #2609

Depends on https://github.com/OpenTreeMap/otm-tiler/pull/109
